time_capture/time_capture/report/project_time/project_time.js

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2016, ALYF GmbH and contributors
+// For license information, please see license.txt
+/* eslint-disable */
+
+frappe.query_reports["Project Time"] = {
+	filters: [
+		{
+			fieldname: "from_date",
+			fieldtype: "Date",
+			label: "From Date",
```

Comment:
```suggestion
			label: __("From Date"),
```

Should we translate the labels in this report?